lib/ReactTags.js

```diff
@@ -308,11 +315,16 @@ class ReactTags extends Component {
     // get the suggestions for the given query
     const query = this.state.query.trim(),
       selectedIndex = this.state.selectedIndex,
-      suggestions = this.state.suggestions,
-      placeholder = this.props.placeholder,
-      inputName = this.props.name,
-      inputId = this.props.id,
-      maxLength = this.props.maxLength;
+      suggestions = this.state.suggestions;
+
+    let {
+      placeholder,
+      name: inputName,
+      id: inputId,
+      maxLength,
+      ...otherProps
+    } = this.props;
+    otherProps = this.removeInternalProps(otherProps);
```

Comment:
lets try not using `let` and do 
`
const modifiedProps = this.removeInternalProps(rest);
`

Reply:
Ok.